autogen/oai/client.py

```diff
@@ -425,6 +430,10 @@ def _register_default_client(self, config: Dict[str, Any], openai_config: Dict[s
                 self._configure_azure_openai(config, openai_config)
                 client = AzureOpenAI(**openai_config)
                 self._clients.append(OpenAIClient(client))
+            elif api_type is not None and api_type.startswith("google"):
+                if GeminiClient is object:
```

Comment:
Maybe something like this instead?
```suggestion
                if GeminiClient.__name__ is == "GeminiClient":
```